### PR TITLE
refactor: Docker-file-simple quickstart

### PR DESCRIPTION
**Pull request by manusa**
## Description
Refactored docker-file-simple (zero-config-dockerfile)
- Renamed to make consistent with rest of docker-file examples
- Removed unnecessary dependencies + tests to speed up build times (quickstarts CI job)
- Added minimum configuration to expose a service
- Made application runnable
- Static file is served from rest endpoint

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] ~I Added [CHANGELOG](../CHANGELOG.md) entry~
 - [ ] ~I have implemented unit tests to cover my changes~
 - [x] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [x] I tested my code in Kubernetes
 - [ ] ~I tested my code in OpenShift~

### CODE REVIEW

File: quickstarts/maven/docker-file-simple/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  7.688 s
+[INFO] Finished at: 2020-06-11T08:01:21+02:00
+[INFO] ------------------------------------------------------------------------
+$ kubectl get pods
+NAME                                      READY   STATUS             RESTARTS   AGE
+docker-file-simple-686cddc6b9-lvgtv       0/1     Running            0          84s
+$ kubectl logs pod/zero-config-dockerfile-686cddc6b9-lvgtv
```
Comment (rohanKanojia) [inline quickstarts/maven/docker-file-simple/README.md:74]: Should we add additional steps for adding `Deployment` and `Service` fragment for port configuration here?

Comment (manusa) [reply quickstarts/maven/docker-file-simple/README.md:74]: I'm not following
Service config is added in pom.xml configuration (36 & 37), this enabled the generation of service and deployment YAML manifests. Do you mean to describe that in the readme too?

Comment (rohanKanojia) [reply quickstarts/maven/docker-file-simple/README.md:74]: Umm, yeah. Does generated `Deployment` manifest contains `ContainerPort`?

Comment (manusa) [reply quickstarts/maven/docker-file-simple/README.md:74]: > Does generated Deployment manifest contains ContainerPort?

Nope, Deployment manifest doesn't contain the containerPort entry(ies). This field is purely [informational](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core), so as the service is already there and quickstart will work either way as expected, maybe adding this extra configurations can be misleading (we already opened https://github.com/eclipse/jkube/issues/240 to take care of this > deployment will be 100% compliant once fixed).

> Umm, yeah.

I'll add a paragraph with the explanation.

**Pull request merged**